docs/user/algo_bc.md

```diff
@@ -0,0 +1,16 @@
+# Behavioral Cloning
```

Comment:
@krzentner can you make sure this covers everything you want to say about BC?

Reply:
Besides the comments I left below, there isn't much more to say.